registry/build.bat

```diff
@@ -1,7 +1,7 @@
 if not exist registry.exe (
   docker build -t registry-builder build
   docker rm -f registry-builder
-  docker run --name registry-builder registry-builder registry --version
+  docker run --name registry-builder registry-builder /go/src/github.com/docker/distribution/registry.exe --version
```

Comment:
As you are here, could you replace the `docker run` with just a `docker create` like in https://github.com/friism/hello-world/blob/32b486472ebcd64fd26b0264dc988958bf44ccf5/update.ps1#L3-L4
We don't have to run any container.